gazebo_color_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
Don't create the plugin in a new package. Put it in `ca_gazebo`.

Reply:
Adressed on the PR referenced below